### PR TITLE
Update rsyslog config to use disk-bufferring and retry when connection to logentries fails

### LINKED ISSUE #30: Rsyslog does not reconnect to logentries on connection failure

### PR DESCRIPTION
**Pull request by lifeeth**
closes #30 #28

### CODE REVIEW

File: src/rsyslog.conf [hunk 1]
```diff
@@ -6,6 +6,11 @@ $ModLoad imklog   # provides kernel logging support
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 
 $WorkDirectory /var/spool/rsyslog
+$ActionQueueFileName fwdRule1     # unique name prefix for spool files
+$ActionQueueMaxDiskSpace 1g       # 1gb space limit (use as much as possible)
```
Comment (Page-) [inline src/rsyslog.conf]: "1gb space limit" and "use as much as possible" seem to conflict?

File: src/rsyslog.conf [hunk 2]
```diff
@@ -6,6 +6,11 @@ $ModLoad imklog   # provides kernel logging support
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 
 $WorkDirectory /var/spool/rsyslog
+$ActionQueueFileName fwdRule1     # unique name prefix for spool files
```
Comment (Page-) [inline src/rsyslog.conf]: It'd be nice to have a more descriptive name, something like `logentries-fwd`

**Comment by Page-** [comment]
Also `bufferring ` -> `buffering` in the commit/changelog please

File: src/rsyslog.conf (hunk 2 above)
Comment (lifeeth) [inline src/rsyslog.conf]: Done!

File: src/rsyslog.conf (hunk 1 above)
Comment (lifeeth) [inline src/rsyslog.conf]: Fixed to reflect this properly.

**Comment by lifeeth** [comment]
Fixed the commit/changelog with the typo

**Comment by lifeeth** [comment]
@Page- Not sure if they can share caches - dont want to mix them both up.

**Pull request merged**